index.html

```diff
@@ -151,18 +151,21 @@
     </p>
 
     <p>
-<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to
-provide verifiable, decentralized digital identity. These new identifiers are
-designed to enable the controller of a <a>DID</a> to prove control over
-it and to be implemented independently of any centralized registry, identity
-provider, or certificate authority. <a>DIDs</a> are URLs that relate a
-<a>DID subject</a> to a <a>DID document</a> allowing trustable interactions with
-that subject. <a>DID documents</a> are simple documents describing how to use
-that specific <a>DID</a>. Each <a>DID document</a> can express cryptographic
-material, verification methods, or <a>service endpoints</a>, which provide a
-set of mechanisms enabling a <a>DID controller</a> to prove control of the
-<a>DID</a>. <a>Service endpoints</a> enable trusted interactions with the
-<a>DID subject</a>.
+<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to 
+provide verifiable, decentralized digital identity. A <a>DID</a> may identify 
+any subject (e.g., a person, organisation, thing, datamodel, abstract entity, etc.)
+that the controller of that <a>DID</a> decides that it 
+identifies. These new identifiers are designed to enable the controller 
+of a <a>DID</a> to prove control over it and to be implemented independently 
+of any centralized registry, identity provider, or certificate authority. 
+<a>DID</a>s are URLs that associate a <a>DID subject</a> with a 
+<a>DID document</a> allowing trustable interactions associated with that subject. 
+Each <a>DID document</a> can express cryptographic material, verification methods, 
+or <a>service endpoints</a>, which provide a set of mechanisms enabling a 
+<a>DID controller</a> to prove control of the <a>DID</a>. <a>Service 
+endpoints</a> enable trusted interactions associated with the <a>DID 
+subject</a>. A <a>DID document</a> may contain semantics about the subject 
+that its identifies, or even the <a>DID subject</a> itself (e.g. a data model).
```

Comment:
Found another typo:

```suggestion
that it identifies, or even the <a>DID subject</a> itself (e.g. a data model).
```

But ... it's the DID that identifies the DID subject anyway, not the DID Document.